zephyr/README.md

```diff
@@ -0,0 +1,33 @@
+## How to setup wolfSSL support for standard Zephyr TLS Sockets and RNG
```

Comment:
Please update the zephyr/include.am to include these with EXTRA_DIST so they are included with a "make dist"

Reply:
So all new files are part of the patch. I dont think the new overlay files should go in EXTRA_DIST as they arent needed, and will not exist, for zephyr users that dont use the patch. Also these new files arent really part of the zephyr wolfssl external module, they are specifically extensions of the zephyr samples so I dont know if it makes sense for extensions of zephyr samples to be in a wolfssl distribution right? Especially since the associated changes in zephyr sockets_tls.c wont necessarily be there, meaning we are then distributing overlays that are dependent on external changes.

Reply:
I can look at trying to do a conditional add but im still not so sure. "Do you want these files part of the wolfssl download package or the commercial release?" I think these changes should be bound to the changes in the zephyr sources. So if those changes to the zephyr sources arent part of the wolfssl download package or commercial release then I believe these should also not be included.

Reply:
It doesn't have to be conditional. Always include as EXTRA_DIST+=....